WooCommerce/Classes/ViewRelated/Products/Edit Product/Product Settings/ProductSettingsRows.swift

```diff
@@ -185,6 +185,37 @@ enum ProductSettingsRows {
         let cellTypes: [UITableViewCell.Type] = [SwitchTableViewCell.self]
     }
 
+    struct DownloadableProduct: ProductSettingsRowMediator {
+         private let settings: ProductSettings
+
+          init(_ settings: ProductSettings) {
+             self.settings = settings
+         }
+
+          func configure(cell: UITableViewCell) {
+             guard let cell = cell as? SwitchTableViewCell else {
+                 return
+             }
+
+              let title = NSLocalizedString("Downloadable Product", comment: "Downloadable Product label in Product Settings")
```

Comment:
Should this go into a `Localization` enum as with other classes? 
I thought that was the standard on the codebase but correct me if I'm wrong!

Reply:
Good idea. I moved all the `NSLocalizationString` of this class under `Localization` enum b71e516738260c44b7d775f455c3f5f7438bafbe